### ISSUE TITLE
/data/raw/onemetre-vaisala generates a server error when the vaisala is offline

### ISSUE THREAD

**Issue by pchote** [warwick-one-metre/dashboard#20]
```
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1517, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.4/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.4/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "./dashboard/__init__.py", line 347, in raw_onemetre_vaisala
    return jsonify(**data)
```

The raw file written by `update-dashboard-data` contains `null`.

**Comment by pchote** [comment]
For now I have manually replaced the json contents with `{}` and commented out the code that updates it.  Will fix it properly when there is more time to test it.

**Comment by pchote** [comment]
Fixed by 6db2c57cd63f7763aa40cb3faff348229c5f0272.
